src/lib/lessons.ts

```diff
@@ -180,6 +180,36 @@ export async function loadLesson(
 // - dash_url - not used
 // - staff_notes_url - not used
 
+export async function loadAssociatedLessonsByTag(tag: string, token?: string) {
+  const graphQLClient = getGraphQLClient(token)
+
+  try {
+    const {lessons} = await graphQLClient.request(
+      loadAssociatedLessonsByTagQuery,
+      {tag},
+    )
+
+    return lessons
+  } catch (e) {
+    throw e
+  }
+}
+
+const loadAssociatedLessonsByTagQuery = /* GraphQL */ `
+  query getAssociatedLessonsByTag($tag: String!) {
+    lessons(tag: $tag, per_page: 20) {
+      id
+      slug
+      completed
+      title
+      description
+      duration
+      free_forever
+      path
+    }
+  }
```

Comment:
a subset of lesson attributes, didn't want to pull every attribute when this is just for the lesson list

grabbing 20 lessons for the list to populate